readme.md

```diff
@@ -12,7 +12,7 @@ Enable AMP on your WordPress site, the WordPress way.
 **License:** [GPLv2 or later](http://www.gnu.org/licenses/gpl-2.0.html)  
 **Requires PHP:** 5.4  
 
-[![Build Status](https://travis-ci.org/ampproject/amp-wp.svg?branch=develop)](https://travis-ci.org/ampproject/amp-wp) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com) 
+[![Build Status](https://travis-ci.org/ampproject/amp-wp.svg?branch=develop)](https://travis-ci.org/ampproject/amp-wp) [![Built with Grunt](https://cdn.gruntjs.com/builtwith.svg)](http://gruntjs.com) 
```

Comment:
This  change should be reverted. By having the latest version of `wp-dev-lib` installed this will be avoided.

Reply:
Looks like I already have the latest version installed. Just triple checked my composer config and vendor folder.

Manually reverted the change now.